content/en/getting_started/synthetics/_index.md

```diff
@@ -22,26 +22,30 @@ further_reading:
 
 ## Overview
 
-Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. Datadog tracks the performance of your webpages and APIs from the backend to the frontend, and at various network levels (`HTTP`, `SSL`, `DNS`, `TCP`, `UDP`, `ICMP`, and `WebSocket`) in a controlled and stable way, alerting you about faulty behavior such as regressions, broken features, high response times, and unexpected status codes.
+Synthetic tests allow you to observe how your systems and applications are performing using **simulated requests and actions from around the globe**. 
 
-There are three ways to monitor your applications: 
+Datadog tracks the performance of your webpages and APIs from the backend to the frontend, and at various network levels (`HTTP`, `SSL`, `DNS`, `TCP`, `UDP`, `ICMP`, and `WebSocket`) in a controlled and stable way. These tests alert you when faulty behaviors like regressions, broken features, high response times, and unexpected status codes happen.
 
-- [API tests][1] to monitor the uptime of your API endpoints
-- [Multistep API tests][2] to link several HTTP requests
-- [Browser tests][3] to test key user journeys
+## Prerequisites
 
-Run all tests from managed locations or from [private locations][4] to monitor internal-facing applications. Synthetic tests can be triggered manually, on a schedule, or directly from your [CI/CD pipelines][5].
+If you haven't already, create a [Datadog account][1].
 
-{{< img src="synthetics/synthetics_home.png" alt="Synthetic Monitoring home page" >}}
+## Synthetic test types
 
-To set up your first Synthetic test with Datadog, follow the sections below.
+To monitor your application, choose a test type: 
 
-## Prerequisites
+- [API tests][2] to monitor the uptime of your API endpoints
+- [Multistep API tests][3] to link several HTTP requests
+- [Browser tests][4] to test key user journeys
 
-If you haven't already, create a [Datadog account][6].
+Run your tests from managed locations or [private locations][5] to monitor any internal-facing applications. Synthetic tests can be triggered manually, on a schedule, or directly from your [CI/CD pipelines][6].
```

Comment:
At first I thought these were links to starting the Getting Started guide, and was confused when they took me into the main docs. It might be better to describe these things in words, without links, and let the Further Reading links at the bottom do the work of helping them find their way to that content? I'm not sure. What do you think?

Reply:
Mmm, I see. I think we can link to the Getting Started with Private Locations page and (eventually) link to the 
 Getting Started with Synthetics and CI/CD page (which is in our scope for OKR #5 in DOCS-3016). 

Should I remove the CI/CD pipelines link for now?